geomesa-spark-jts/src/test/scala/org/locationtech/geomesa/spark/SparkSQLGeometricCastTest.scala

```diff
@@ -8,34 +8,30 @@
 
 package org.locationtech.geomesa.spark
 
-import java.util.{Map => JMap}
 
-import com.typesafe.scalalogging.LazyLogging
-import com.vividsolutions.jts.geom.{LineString, Polygon, Point, Geometry}
-import org.apache.spark.sql._
-import org.geotools.data.{DataStoreFinder, DataStore}
+import com.vividsolutions.jts.geom.{LineString, Point, Polygon}
+import org.apache.spark.sql.{SQLContext, SparkSession}
+import org.apache.spark.sql.jts.SQLTypes
 import org.junit.runner.RunWith
-import org.locationtech.geomesa.utils.interop.WKTUtils
 import org.specs2.mutable.Specification
 import org.specs2.runner.JUnitRunner
 
-import scala.collection.JavaConversions._
 
 @RunWith(classOf[JUnitRunner])
-class SparkSQLGeometricCastTest extends Specification with LazyLogging {
+class SparkSQLGeometricCastTest extends Specification {
 
   "sql geometry accessors" should {
     sequential
 
-    val dsParams: JMap[String, String] = Map("cqengine" -> "true", "geotools" -> "true")
-    var ds: DataStore = null
     var spark: SparkSession = null
     var sc: SQLContext = null
 
     // before
     step {
-      ds = DataStoreFinder.getDataStore(dsParams)
-      spark = SparkSQLTestUtils.createSparkSession()
+      spark = SparkSession.builder()
+        .appName("testSpark")
+        .master("local[*]")
+        .getOrCreate()
       sc = spark.sqlContext
       SQLTypes.init(sc)
```

Comment:
Because of this line, I'd suggest renaming `SQLTypes` to something more specific to this being associated with JTS. With all the imports involved, just calling it `SQLTypes` doesn't give hints to new users as to what it's for.

Personally, I prefer the extension method approach to injecting this kind of behavior. Here's the RasterFrames version of the same:

https://github.com/s22s/raster-frames/blob/54b8b74f6b1971ed0bbbe3ba5696c932d8f8d3fa/src/main/scala/astraea/spark/rasterframes/SQLContextMethods.scala#L32-L40